pkg/table/tables/tables_test.go

```diff
@@ -488,7 +488,7 @@ func TestHiddenColumn(t *testing.T) {
 	tk.MustGetErrMsg("select d, b from t;", "[planner:1054]Unknown column 'd' in 'field list'")
 	tk.MustGetErrMsg("select * from t where b > 1;", "[planner:1054]Unknown column 'b' in 'where clause'")
 	tk.MustGetErrMsg("select * from t order by b;", "[planner:1054]Unknown column 'b' in 'order clause'")
-	tk.MustGetErrMsg("select * from t group by b;", "[planner:1054]Unknown column 'b' in 'group statement'")
+	tk.MustGetErrMsg("select sum(a) from t group by b;", "[planner:1054]Unknown column 'b' in 'group statement'")
```

Comment:
It's changed. `select * from t group by b;` will return `Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column`, because we changed the order of checking `ONLY_FULL_GROUP_BY` and `rewrite`.

Reply:
can we check a little bit of what the error will be in mysql

Reply:
okk